src/Home.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import getToken from './lib/tokens';
+import { useHistory } from 'react-router-dom';
+import { firestore } from './lib/firebase';
+
+const Home = () => {
+  const history = useHistory();
+
+  useEffect(() => {
+    const token = JSON.parse(localStorage.getItem('token'));
+    if (token) history.push('/list');
+  });
+
+  const createToken = async () => {
+    const token = getToken();
+    localStorage.setItem('token', JSON.stringify(token));
```

Comment:
Curious about this line. What was the reason behind using `JSON.stringify`? 

Asking because I noticed that the value stored in `localStorage` includes the quotation marks. Which isn't a huge deal either way, but means that the team will have to be consistent about always including the quotation marks when checking tokens down the road.

Reply:
Now I can't remember why we did that.... I'm going to modify to take it out so we don't run into that consistency issue later on!